README.rst

```diff
@@ -37,6 +37,32 @@ repository name, and an optional log level:
 
     $ farcy --level INFO appfolio farcy
 
+Configuration
+~~~~~~~~~~~~~
+
+Farcy allows to be configured using configuration files. Existence of a configuration
+file is optional and values can be overwritten by commandline arguments. On boot,
+Farcy will look for a configuration file at ``~/.config/farcy/farcy.conf``.
+
+The configuration file can contain a ``DEFAULT`` section and a section with repository
```

Comment:
Should we indicate that more than one repository specific setting can be provided?